### DISCUSSION TITLE
Is `PT014` implementable?

### DISCUSSION THREAD

**Discussion by harupy** [astral-sh/ruff#6597] [Q&A] [answered]
Is [PT014](https://github.com/m-burst/flake8-pytest-style/blob/v1.7.2/docs/rules/PT014.md) implementable?

**Comment by harupy** [comment]
https://github.com/astral-sh/ruff/pull/1506

> Won't implement PT014 for now. It requires checking for equivalent nodes, which is a bit more complicated

Is this still the case?

**Reply by charliermarsh** [reply, marked as answer]
We can do that now -- see `ComparableExpr` which lets you compare nodes ignoring their row and column number.

**Reply by harupy** [reply]
@charliermarsh Thanks for the quick reply!

**Reply by harupy** [reply]
[B033](https://beta.ruff.rs/docs//rules/duplicate-value) looks similar.
